<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-results xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="nunit_schema_2.5.xsd" name="Pester" total="143" errors="0" failures="130" not-run="0" inconclusive="0" ignored="0" skipped="2" invalid="0" date="2023-03-02" time="11:51:06">
  <environment nunit-version="2.5.8.0" machine-name="AU7134QV2" clr-version="Unknown" user="squires" platform="Microsoft Windows 10 Education|C:\WINDOWS|\Device\Harddisk0\Partition3" os-version="10.0.19045" cwd="C:\Scripts\Repositories\PSRocketLaunch\tests" user-domain="AMERICAN" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="TestFixture" name="Pester" executed="True" result="Failure" success="False" time="2.8167" asserts="0" description="Pester">
    <results>
      <test-suite type="TestFixture" name="C:\Scripts\Repositories\PSRocketLaunch\tests\Help.tests.ps1" executed="True" result="Failure" success="False" time="1.6496" asserts="0" description="C:\Scripts\Repositories\PSRocketLaunch\tests\Help.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.1929" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.2587" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.361" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.4602" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.5485" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.6236" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0.6825" asserts="0" description="Test help for &lt;_.Name&gt;">
            <results>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;" executed="True" result="Failure" success="False" time="0" asserts="0" description="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;">
                <results>
                  <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct [mandatory] value" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct [mandatory] value" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Has correct parameter type" name="Test help for &lt;_.Name&gt;.Parameter &lt;_.Name&gt;.Has correct parameter type" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case description="Help is not auto-generated" name="Test help for &lt;_.Name&gt;.Help is not auto-generated" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has description" name="Test help for &lt;_.Name&gt;.Has description" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example code" name="Test help for &lt;_.Name&gt;.Has example code" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-case description="Has example help" name="Test help for &lt;_.Name&gt;.Has example help" time="0" asserts="0" success="False" result="Failure" executed="True">
                <failure>
                  <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                  <stack-trace />
                </failure>
              </test-case>
              <test-suite type="ParameterizedTest" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid" executed="True" result="Failure" success="False" time="0" asserts="0" description="Help link &lt;_&gt; is valid">
                <results>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case description="Help link &lt;_&gt; is valid" name="Test help for &lt;_.Name&gt;.Help link &lt;_&gt; is valid()" time="0" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>This test should run but it did not. Most likely a setup in some parent block failed.</message>
                      <stack-trace />
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="C:\Scripts\Repositories\PSRocketLaunch\tests\Manifest.tests.ps1" executed="True" result="Ignored" success="True" time="0.7669" asserts="0" description="C:\Scripts\Repositories\PSRocketLaunch\tests\Manifest.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Module manifest" executed="True" result="Success" success="True" time="0.4593" asserts="0" description="Module manifest">
            <results>
              <test-suite type="TestFixture" name="Module manifest.Validation" executed="True" result="Success" success="True" time="0.4478" asserts="0" description="Module manifest.Validation">
                <results>
                  <test-case description="Has a valid manifest" name="Module manifest.Validation.Has a valid manifest" time="0.1817" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid name in the manifest" name="Module manifest.Validation.Has a valid name in the manifest" time="0.0433" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid root module" name="Module manifest.Validation.Has a valid root module" time="0.0295" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid version in the manifest" name="Module manifest.Validation.Has a valid version in the manifest" time="0.022" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid description" name="Module manifest.Validation.Has a valid description" time="0.0264" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid author" name="Module manifest.Validation.Has a valid author" time="0.0243" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid guid" name="Module manifest.Validation.Has a valid guid" time="0.0483" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid copyright" name="Module manifest.Validation.Has a valid copyright" time="0.023" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Has a valid version in the changelog" name="Module manifest.Validation.Has a valid version in the changelog" time="0.0217" asserts="0" success="True" result="Success" executed="True" />
                  <test-case description="Changelog and manifest versions are the same" name="Module manifest.Validation.Changelog and manifest versions are the same" time="0.02" asserts="0" success="True" result="Success" executed="True" />
                </results>
              </test-suite>
            </results>
          </test-suite>
          <test-suite type="TestFixture" name="Git tagging" executed="True" result="Ignored" success="True" time="0.4918" asserts="0" description="Git tagging">
            <results>
              <test-case description="Is tagged with a valid version" name="Git tagging.Is tagged with a valid version" time="0.0214" asserts="0" success="False" result="Ignored" executed="False" />
              <test-case description="Matches manifest version" name="Git tagging.Matches manifest version" time="0.008" asserts="0" success="False" result="Ignored" executed="False" />
            </results>
          </test-suite>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="C:\Scripts\Repositories\PSRocketLaunch\tests\Meta.tests.ps1" executed="True" result="Failure" success="False" time="0.4002" asserts="0" description="C:\Scripts\Repositories\PSRocketLaunch\tests\Meta.tests.ps1">
        <results>
          <test-suite type="TestFixture" name="Text files formatting" executed="True" result="Failure" success="False" time="0.095" asserts="0" description="Text files formatting">
            <results>
              <test-suite type="TestFixture" name="Text files formatting.File encoding" executed="True" result="Success" success="True" time="0.029" asserts="0" description="Text files formatting.File encoding">
                <results>
                  <test-case description="No text file uses Unicode/UTF-16 encoding" name="Text files formatting.File encoding.No text file uses Unicode/UTF-16 encoding" time="0.0262" asserts="0" success="True" result="Success" executed="True" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="Text files formatting.Indentations" executed="True" result="Failure" success="False" time="0.0923" asserts="0" description="Text files formatting.Indentations">
                <results>
                  <test-case description="No text file use tabs for indentations" name="Text files formatting.Indentations.No text file use tabs for indentations" time="0.0602" asserts="0" success="False" result="Failure" executed="True">
                    <failure>
                      <message>Expected 0, but got 1.</message>
                      <stack-trace>at $totalTabsCount | Should -Be 0, C:\Scripts\Repositories\PSRocketLaunch\tests\Meta.tests.ps1:47
at &lt;ScriptBlock&gt;, C:\Scripts\Repositories\PSRocketLaunch\tests\Meta.tests.ps1:47</stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>